data/src/main/java/pl/edu/agh/model/Sprint.java

```diff
@@ -1,5 +1,48 @@
 package pl.edu.agh.model;
 
+import java.time.Duration;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Transient;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
 public class Sprint {
 
+    // TODO: use a base class for id column
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(nullable = false)
+    private Long id;
+
+    // TODO: hook up reference
+    @Transient
+    private Project project;
+
+    private LocalDateTimePeriod durationPeriod;
+    private LocalDateTimePeriod enrollmentPeriod;
+
+    @Column(nullable = false)
+    private boolean closed;
+
+    private Duration timeWorked;
```

Comment:
`timeBurned` + `timePlanned`?

Reply:
czas przepracowany -- `timeBurned`
czas potrzebny na taski w sumie na poczatku sprintu -- `timePlanned`
czas potrzebny na taski w sumie na koncu sprintu -- czy z tego zrezygnowalismy?

Reply:
Jak dla mnie `timePlanned` jest podany też na końcu sprintu - można zrobić tak, żeby był podawany wcześniej i update'owany, ale to chyba generalnie jest jedna liczba